homeassistant/components/workday/strings.json

```diff
@@ -16,17 +16,17 @@
         "data": {
           "excludes": "Excludes",
           "days_offset": "Offset",
-          "workdays": "Workdays",
+          "workdays": "Includes",
           "add_holidays": "Add holidays",
           "remove_holidays": "Remove Holidays",
           "province": "Subdivision of country",
           "language": "Language for named holidays",
           "category": "Additional category as holiday"
         },
         "data_description": {
-          "excludes": "List of workdays to exclude, notice the keyword `holiday` and read the documentation on how to use it correctly",
+          "excludes": "List of weekdays to exclude, plus the keyword `holidays` (read the documentation on how to use it correctly)",
```

Comment:
Instead of writing `read the docs` on both these lines maybe we should add a description to the step to overall guide the user to read the docs?

Reply:
No problem, it's certainly best to explain it in the dialog and only link to the docs as a last resort.
How about:

- List of all weekdays plus `holidays` according to the settings below to include as possible workdays.
- List of all weekdays plus `holidays` according to the settings below to exclude as workdays.

I'm not sure why `holidays` is also an option for the Include list, but as long as it's available there we need to explain it for both fields.

Reply:
As including `holidays` as possible workdays does negate the sense of the whole integration we could leave this out from the first description and simplify as follows:

- Select which weekdays to include as possible workdays.
- Select which weekdays to exclude as workdays. The key `holidays` adds all days according to the settings below.

I dropped the word "list" as the UI does not show a list in the very sense of that word.

Reply:
Well, the key `holiday` also includes a lot of dates depending on which country/province you set so I still think we should probably refer to the documentation and not only to the include/exclude below.

Reply:
The line "The key `holidays` adds all days according to the settings below." is located _under_ the Exclude (and thus also below the Include fields) so it points to the country and custom settings that come next in the dialog.

So in that context I think that it's clear that the settings that follow control what is evaluated as a holiday.

Reply:
As mentioned I removed the explanation for holidays from the Includes field as adding this is an edge case that won't be used in 99.9 % of all setups.

This way the "below" you mentioned is used only on the Exclude description and points to the detailed country settings and customizations that come next in the dialog.